docs/style-guide.md

```diff
@@ -44,8 +44,8 @@ const fs = Promise.promisifyAll(require('fs-extra'))
 
 ### Leave a blank line (EOL) at the end of file
 
-### Comments
-Feel free to leave comments where necessary.
+### Code Comments
+Feel free to comment your code when necessary.
```

Comment:
Yes I know, it is a bit confusing when I read it first, so I changed the wording in an attempt to enhance the clarity. We can definitely make it better.